src/const.ts

```diff
@@ -233,6 +243,10 @@ export interface Constant extends Required<ConstantProps> {
    * @private Callback to be called when the widget open state changes.
    */
   onWidgetOpenStateChange?: (params: OnWidgetOpenStateChangeParams) => void;
+  /**
+   * @private Message input state controlling properties.
+   */
+  messageInputControlProps?: MessageInputControlProps;
```

Comment:
How about avoiding the use of the term "Props" in contexts where it refers to parameters of general functions or specific options, since in a React development environment, it specifically means attributes passed to a component?

Reply:
I see. Do you have any suggestions? I thought of messageInputControlAttributes.

Reply:
`messageInputControls` or `inputControls`?

Personally, I'm not sure if there will be additional options to control the input in the future, so I prefer to avoid nested objects.

Reply:
Yep we can try avoid nested object interface. I think `disabled` is likely to be requested in the future. 
```
MessageInputControls {
  // disabled?: boolean; could be added in the future.
  blockWhileBotResponding?: boolean | number;
}
```

Reply:
Sure, that sounds make sense. Additionally, we would like to receive the placeholder text for each case where it will be disabled in the future.